### PR TITLE
sklearn deprecation, remove `normalize` from LassoLars

### PR DESCRIPTION
**Pull request by kushalkolar**
# Description

minor change, the latest sklearn v1.2 that just released deprecated the `normalize` kwarg for `LassoLars`, it just produces a massive stream of warnings during CNMF (they can be disabled but it'll be removed by v1.4 so might as well fix it?)

```python3
/home/kushalk/python-venvs/mescore/lib/python3.10/site-packages/sklearn/linear_model/_base.py:117: FutureWarning: 'normalize' was deprecated in version 1.2 and will be removed in 1.4. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.
  warnings.warn(
```

not exactly sure which tests should be used, I ran `nose2` in the root dir with both `nose` and `nose2` installed and the tests pass, see #1021

### CODE REVIEW

**Comment by pgunn** [comment]
Thanks!
